pkg/storage/system_intake_test.go

```diff
@@ -129,7 +103,7 @@ func (s StoreTestSuite) TestSaveSystemIntake() {
 
 func (s StoreTestSuite) TestFetchSystemIntakeByID() {
 	s.Run("golden path to fetch a system intake", func() {
-		intake := newSystemIntake()
+		intake := testhelpers.NewSystemIntake()
 		id := intake.ID
 		tx := s.db.MustBegin()
 		_, err := tx.NamedExec("INSERT INTO system_intake (id, eua_user_id, status) VALUES (:id, :eua_user_id, :status)", &intake)
```

Comment:
It looks like in the business case tests we're using the Fetch and Save storage methods to set up tests, where here it looks like we're still rewriting queries with tx.NamedExec. Can we normalize this? (I think I prefer using the storage methods everywhere)

Reply:
I agree. I think we should normalize this in another pr, though after golive, if that's ok with you.